Microsoft.Packaging.Tools/tasks/Trimming/NuGetPackageNode.cs

```diff
@@ -28,6 +28,7 @@ public NuGetPackageNode(string id, string version, IEnumerable<string> dependenc
         public bool IsIncluded { get; set; }
         public string Version { get; }
         public bool IsProject { get; }
+        public bool IsMetaPackage { get { return Files.Count == 0; } }
```

Comment:
Meta-package is actually a cross-technology term that means a package that only has dependencies.  Linux uses the same term as do other packaging technologies.  As such I believe there is precedent for calling this a meta-package.

I don't think it's right to call our current *big* packages meta-packages.  Instead I'd call them  framework or platform packages.